utils/create-module.js

```diff
@@ -14,7 +14,7 @@ const addModuleAsDependencyByType = (cwd, moduleName, type) => {
   const json = JSON.parse(packageContents);
 
   // Add module, sort dependencies, then write back out
-  json.dependencies[`@workday/canvas-kit-${type}-${moduleName}`] = `^${json.version}`;
+  json.dependencies[`@workday/canvas-kit-${type}-${moduleName}`] = `0.0.0`;
```

Comment:
Thanks @NicholasBoll I did mean to come back and do this but thanks for pulling it through and best of luck :)